interface/github.py

```diff
@@ -0,0 +1,58 @@
+"""Utility classes for interacting with Github API via PyGithub."""
+from github import Github, GithubException
+
+
+class GitBot:
```

Comment:
I'm not really a fan of the name `GitBot` - this doesn't really have anything to do with being a bot (even though it is consistent with `SlackBot`). I'd prefer both this and the Slack class get renamed - honestly, even `Slack` and `Github` might be okay, assuming they're in the `interface` module.

Reply:
haha ok

Reply:
Naming it github might be confusing since the pygithub class is also called github, is GitInterface ok?